rollup.config.js

```diff
@@ -1,21 +1,29 @@
-import commonjs from 'rollup-plugin-commonjs';
-import nodeBuiltins from 'rollup-plugin-node-builtins';
-import nodeResolve from 'rollup-plugin-node-resolve';
-import babel from 'rollup-plugin-babel';
+'use strict';
 
-export default {
+const commonjs = require('rollup-plugin-commonjs');
+const nodeGlobals = require('rollup-plugin-node-globals');
+const nodeBuiltins = require('rollup-plugin-node-builtins');
+const nodeResolve = require('rollup-plugin-node-resolve');
+const jsonPlugin = require('rollup-plugin-json');
+const babel = require('rollup-plugin-babel');
+const scenariosPlugin = require('./tools/scenarios-plugin');
+
+module.exports = {
   input: 'index.js',
   output: {
     file: 'dist/bson.js',
     format: 'umd',
-    name: 'bson',
+    name: 'bson'
   },
   plugins: [
+    scenariosPlugin(),
```

Comment:
Makes sense! We made a modification so that the scenariosPlugin is only added to the plugins array in the karma config, so the rollup config is essentially left unchanged.